### PR TITLE
TAP 1.9 buildpack/stack docs

### PR DESCRIPTION
**Pull request by thitch97**
# Which other branches do you want a technical writer to cherry-pick this PR to (if any)?
This change only applies to 1.9. Part of pivotal-cf/tanzu-buildpacks#482.

It's best to PR to the most recent relevant branch and leave all the cherry-picking to the
writer, except where earlier branches require factual changes to the content.
For more information about the branches, see https://github.com/pivotal/docs-tap#branches

### CODE REVIEW

**Comment by thitch97** [comment]
@johnnyr0x

File: tanzu-build-service/dependencies.hbs.md [hunk 1]
```diff
@@ -148,25 +148,25 @@ The `full` dependencies contain the following buildpacks in Tanzu Application Pl
 |-----------|---------|------------------|
 | Java Buildpack for VMware Tanzu | 9.15.2 | Bionic, Jammy, UBI |
 | Java Native Image Buildpack for Tanzu | 7.13.2 | Bionic, Jammy |
```
Comment (johnnyr0x) [inline tanzu-build-service/dependencies.hbs.md:150]: Java: 9.16.0
Java Native Image: 7.14.0

File: tanzu-build-service/dependencies.hbs.md [hunk 2]
```diff
@@ -95,25 +95,25 @@ in the VMware Tanzu Buildpacks documentation
 
 #### <a id="lite-buildpacks"></a> Lite dependencies: buildpacks
 
-The `lite` dependencies contain the following buildpacks in Tanzu Application Platform v1.8:
+The `lite` dependencies contain the following buildpacks in Tanzu Application Platform v1.9:
 
 | Buildpack | Version | Supported Stacks |
 |-----------|---------|------------------|
 | Java Buildpack for VMware Tanzu (Lite) | 9.15.2 | Bionic, Jammy, UBI |
 | Java Native Image Buildpack for Tanzu (Lite) | 7.13.2 | Bionic, Jammy |
```
Comment (johnnyr0x) [inline tanzu-build-service/dependencies.hbs.md:103]: Java (lite) : 9.16.0
Java Native Image (lite) : 7.14.0

**Review by johnnyr0x — COMMENTED** [review]
Java versions need an update

File: tanzu-build-service/dependencies.hbs.md [hunk 3]
```diff
@@ -99,8 +99,8 @@ The `lite` dependencies contain the following buildpacks in Tanzu Application Pl
 
 | Buildpack | Version | Supported Stacks |
 |-----------|---------|------------------|
-| Java Buildpack for VMware Tanzu (Lite) | 9.15.2 | Bionic, Jammy, UBI |
-| Java Native Image Buildpack for Tanzu (Lite) | 7.13.2 | Bionic, Jammy |
+| Java Buildpack for VMware Tanzu (Lite) | 9.16.0 | Bionic, Jammy, UBI |
+| Java Native Image Buildpack for Tanzu (Lite) | 7.14.0 | Bionic, Jammy |
 | .NET Core Buildpack for VMware Tanzu (Lite) | 2.13.0 | Bionic, Jammy |
```
Comment (johnnyr0x) [inline tanzu-build-service/dependencies.hbs.md:104]: | .NET Core Buildpack for VMware Tanzu (Lite) | 2.13.0 | Bionic, Jammy, UBI |

Comment (johnnyr0x) [reply tanzu-build-service/dependencies.hbs.md:104]: | Web Servers Buildpack for VMware Tanzu (Lite) | 0.19.0 | Bionic, Jammy, UBI |

**Review by johnnyr0x — COMMENTED** [review]
Webservers and .NET have UBI

**Pull request merged**